packages/unicode_emoji.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Unicode_emoji < Package
```

Comment:
Ah yes, the all important emoji. :)

Reply:
Sadly, it was easier to make these packages happen than try to figure out how to make `ibus` compile without them,